gateway-ha/src/main/java/io/trino/gateway/ha/config/DataStoreConfiguration.java

```diff
@@ -20,14 +20,16 @@ public class DataStoreConfiguration
     private String password;
     private String driver;
     private Integer queryHistoryHoursRetention = 4;
+    private boolean runMigrationsEnabled;
```

Comment:
This could be defaulted to `true`. Most users should allow the gateway to manage their DB schema, and it makes things easier for new users as well.

Reply:
Please document this setting

Reply:
I think https://github.com/trinodb/trino-gateway/blob/f4b78f48ca5e882575a4f82640bb70940195c79b/docs/installation.md?plain=1#L39 is probably the best place for it now. You can revise the section that refers to initializing the DB manually and let users know that the Gateway will do it for them, unless they set `runMigrationsEnabled: false`, in which case they will need to manually run the schema SQLs